ci_framework/roles/reproducer/tasks/rotate_log.yml

```diff
@@ -0,0 +1,12 @@
+---
+- name: "Check {{ item }}"
+  register: log_file
+  ansible.builtin.stat:
+    path: "{{ item }}"
+
+- name: Rotate log if present
+  when:
+    - log_file.stat.exists
+  ansible.builtin.shell:
```

Comment:
Is `ansible.builtin.command` not working for this `mv`?

Reply:
ok, there's actually a better way I didn't know.
Note that, apparently, there's a fact available in ansible, created during the fact gathering, named `ansible_date_time` as described [here](https://www.ansiblepilot.com/articles/using-date-time-and-timestamp-in-ansible-playbook-ansible-tip-and-tricks/).

But since this timestamp isn't updated, it may be better to consume a fresh one, using the method I introduced (setting an internal var calling a `lookup('pipe', 'date +%s')` ).